### ISSUE TITLE
Add sprite.play_animation

### ISSUE THREAD

**Issue by subsoap** [defold/editor2-issues#878]
Another convenience thing for autocomplete. I think this was suggested before on the forum a long time ago but not here.

```
sprite.play_animation(url, animation) -- would probably be better

sprite.play_animation(".", "jump") -- don't require hash, "." sends to all sprite
sprite.play_animation("#sprite", "jump") -- only sent to #sprite component of GO
```

These would still be a proxy for the normal way
```
msg.post("#sprite", "play_animation", {id = hash("jump")})
```

**Comment by Malmer** [comment]
If you want it, just can just add it yourself at startup to the sprite table:

```
function sprite.play_animation(url, id)
  msg.post(url, "play_animation", {id = hash(id)})
end
```

**Comment by subsoap** [comment]
I'm thinking that making a module which adds all of these helper functions which should exist would be a good idea, but they really should be a part of the standard too. Every msg.post function should be considered if an autocomplete helper version can be made standard. msg.post is REALLY great, but it sucks that it can't be autocompleted in a smart way because I forget the names of things so often especially when working with multiple languages over several proejcts at once and so have to look them up often.

**Comment by AGulev** [comment]
@erikangelin-king @subsoap  Solved in Defold 1.2.130
https://www.defold.com/ref/sprite/#sprite.play_flipbook:url-id--complete_function-
https://forum.defold.com/t/defold-1-2-130-has-been-released/22487?u=agulev
